tests/unit_tests/test_wave_06.py

```diff
@@ -84,91 +76,78 @@ def test_best_by_category_with_duplicates():
     assert best_item.get_category() == "Clothing"
     assert best_item.condition == pytest.approx(4.0)
 
-@pytest.mark.skip
+
+# @pytest.mark.skip
 def test_swap_best_by_category():
     # Arrange
     # me
     item_a = Decor(condition=2.0)
     item_b = Electronics(condition=4.0)
     item_c = Decor(condition=4.0)
-    tai = Vendor(
-        inventory=[item_a, item_b, item_c]
-    )
+    tai = Vendor(inventory=[item_a, item_b, item_c])
 
     # them
     item_d = Clothing(condition=2.0)
     item_e = Decor(condition=4.0)
     item_f = Clothing(condition=4.0)
-    jesse = Vendor(
-        inventory=[item_d, item_e, item_f]
-    )
+    jesse = Vendor(inventory=[item_d, item_e, item_f])
 
     # Act
     result = tai.swap_best_by_category(
-        other_vendor=jesse,
-        my_priority="Clothing",
-        their_priority="Decor"
+        other_vendor=jesse, my_priority="Clothing", their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That the results is truthy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    assert result == True
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert tai.inventory == [item_a, item_b, item_f]
+    assert jesse.inventory == [item_d, item_e, item_c]
```

Comment:
Be careful using `==` for asserts on lists! This requires that all of the items in both lists are in the exact same order. It would be safer to use `in` like the other tests in this file.